workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -659,7 +659,7 @@ pre[class*='language-'] {
 	.form-group-item select {
 		align-items: center;
 		/* stylelint-disable-next-line function-url-quotes */
-		background: url("data:image/svg+xml;utf-8,<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24'><path d='M7 10l5 5 5-5z'/><path fill='rgba(11, 95, 255, 1)' d='M7 10l5 5 5-5z'/></svg>")
+		background-color: url("data:image/svg+xml;utf-8,<svg xmlns='http://www.w3.org/2000/svg' width='24' height='24'><path d='M7 10l5 5 5-5z'/><path fill='rgba(11, 95, 255, 1)' d='M7 10l5 5 5-5z'/></svg>")
```

Comment:
background-color cannont take a url, it should be a background property if we are using a background image see [this](https://www.w3schools.com/cssref/css3_pr_background.php) for more info

Reply:
Done!